cfme/scripting/quickstart.py

```diff
@@ -65,7 +65,7 @@ def args_for_current_venv():
     DEBUG_INSTALL_COMMAND = 'dnf debuginfo-install -y'
 elif HAS_YUM:
     INSTALL_COMMAND = 'yum install -y'
-    DEBUG_INSTALL_COMMAND = 'yum debuginfo-install -y'
+    DEBUG_INSTALL_COMMAND = 'debuginfo-install -y'
```

Comment:
In F27, I'm not seeing `debugino-install` as an available command without yum-utils installed. I think this is going to require adding yum-utils to the Fedora releases as well so they can run it directly without `yum`

Reply:
I should add a comment.   It won't be a problem in F27 because it uses DNF.